src/main/scala/temple/generate/server/go/service/main/GoServiceMainCreateHandlerGenerator.scala

```diff
@@ -74,7 +67,8 @@ object GoServiceMainCreateHandlerGenerator {
         mkCode.doubleLines(
           when(root.projectUsesAuth) { generateExtractAuthBlock(usesVar = true) },
           generateDecodeRequestBlock(s"create${root.name}"),
-          generateRequestNilCheck(root, clientAttributes),
+          // TODO: Handle this properly, there could be serverSet attributes
+          when(clientAttributes.nonEmpty) { generateRequestNilCheck(root, clientAttributes) },
```

Comment:
What’s happening with this TODO?

Reply:
See comments in Jay's review: https://github.com/TempleEight/temple/pull/253#discussion_r404765886

This is a temporary fix to make the generated Go valid